src/ontology/edits/oeo-model.omn

```diff
@@ -106,12 +106,18 @@ ObjectProperty: OEO_00000503
 ObjectProperty: OEO_00000511
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A relation that holds between an optimization and its objective function or objective variable."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A relation that holds between an optimisation and its objective function or objective variable."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "correct spelling optimisation and add range 'objective variable'
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/848
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/878",
         rdfs:label "has objective"
     
     Domain: 
         OEO_00000313
     
+    Range: 
+        OEO_00240004
```

Comment:
```suggestion
        OEO_00000304 or OEO_00240004
```

The range of `has objective` includes both the `objective function` and the `objective variable`.

Reply:
Sorry, I misunderstood you.
I hope, it is correct now.